runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsPlugin.java

```diff
@@ -233,6 +227,10 @@ private void startPrayerDrainTimer()
 	{
 		if (config.showPrayerDrainTimer())
 		{
+			if (barrowsPrayerDrainTimer != null)
```

Comment:
Why is this added? The only place which calls this already tests that this is null. You could maybe assert this if you really wanted.

Reply:
I explained in the description, that in case you didn't want to change the logic that determined when you start or stop the timer, that this would also prevent the boxes from stacking up.  So I made each "solution" a separate commit to easily drop whichever one you'd least prefer.